### ISSUE TITLE
Update dependencies for vulnerability alerts only

### ISSUE THREAD

**Issue by Waldo000000** [renovatebot/config-help#138]
Is there a way to configure renovate to raise PRs for [vulnerability alerts](https://renovatebot.com/docs/configuration-options/#vulnerabilityalerts) only? I've tried this:
```
{
  "extends": [
    "config:base"
  ],
  "enabled": "false",
  "vulnerabilityAlerts": {
    "enabled": "true"
  }
}
```
But from the logs that results in `Repository is disabled - skipping`.

**Comment by Waldo000000** [comment]
Actually, apparently I haven't managed to achieve this. It seems that renovate is raising PRs out of schedule (or I've misunderstood how the schedule works), so I'm still looking for an alternative. See e.g. below:

![2019-01-10 10_40_01-update dependency types_clipboard to v2 by renovate pull request 655 octop](https://user-images.githubusercontent.com/1407488/50938491-35301580-14c4-11e9-8507-9297ee0f8759.png)

**Comment by rarkins** [comment]
Hi,

Can you try this?

```
{
  "packageRules": [
    "packagePatterns": ["*"],
    "enabled": false
  ],
  "vulnerabilityAlerts": {
    "enabled": "true"
  }
}
```

**Comment by Waldo000000** [comment]
Thanks @rarkins. I've made that update and will see what happens.

**Comment by prestonvanloon** [comment]
@rarkins, we've also made the same change here: https://github.com/prysmaticlabs/website/blob/master/renovate.json

If anyone is looking for an example ^

**Comment by Waldo000000** [comment]
@rarkins I've tried that out (as we've recently had a GitHub alert pop up) and this was the result:

1. Initially we had a `schedule:monthly` preset, and this suppressed the PR (it was there just in case, in future, we decide to enable PRs for some other packages)
2. I removed the `schedule:monthly` preset, and this resulted in a PR being raised to address the vulnerability :tada: However, it just looked like a regular renovate PR and was missing the `commitMessageSuffix`.

Are those two above behaviors expected?
